base_layer/core/src/chain_storage/lmdb_db/lmdb_db.rs

```diff
@@ -2020,6 +2020,9 @@ impl BlockchainBackend for LMDBDatabase {
                 //  lmdb_len(&txn, &self.utxo)
                 unimplemented!("Need to get rangeproof mmr size")
             },
+            MmrTree::Input => {
+                unimplemented!("Need to get input mmr size")
+            },
```

Comment:
are these unused?

Reply:
Yeah you used to be able to call them from the outside - but not now -but yes, this is not kosher - fixed :)